src/app/seats/seats.component.html

```diff
@@ -67,25 +67,17 @@
     </div>
     <ng-template #allShows>
       <div class="shows">
-        <div
-          class="showtime"
-          *ngFor="let showDetails of shows; index as Arrind"
-        >
-          <label>{{ showDetails.hall_name | uppercase }}</label
-          ><br />
-          <button
-            mat-raised-button
-            color="primary"
-            [value]="availability.show_id"
-            *ngFor="
+        <div class="showtime" *ngFor="let showDetails of shows; index as Arrind">
```

Comment:
What is "Arrind"?
does this word have any meaning?
this is not following any variable name convention.

Follow camelCase everyewhere.

Reply:
changed to camelcase-- "hallIndex"